src/main/target/TBS_LUCID_FC/target.h

```diff
@@ -0,0 +1,151 @@
+/*
+ * This file is part of INAV Project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Alternatively, the contents of this file may be used under the terms
+ * of the GNU General Public License Version 3, as described below:
+ *
+ * This file is free software: you may copy, redistribute and/or modify
+ * it under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This file is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER   "LUFC"
+
+#define USBD_PRODUCT_STRING       "TBS Lucid FC"
+
+#define LED0                      PC14
+#define LED1                      PC15
+
+#define BEEPER_INVERTED
+#define BEEPER                    PC13
+
+#define USE_VCP
+
+#define USE_UART1
+#define UART1_TX_PIN              PA9
+#define UART1_RX_PIN              PA10
+
+#define USE_UART2
+#define UART2_TX_PIN              NONE
+#define UART2_RX_PIN              PB0
+
+#define USE_UART3
+#define UART3_TX_PIN              PB11
+#define UART3_RX_PIN              PB10
+
+#define USE_UART4
+#define UART4_TX_PIN              PH3
+#define UART4_RX_PIN              PH2
+
+#define USE_UART5
+#define UART5_TX_PIN              PB9
+#define UART5_RX_PIN              PD2
+
+#define USE_UART7
+#define UART7_TX_PIN              PB4
+#define UART7_RX_PIN              PB3
+
+#define USE_UART8
+#define UART8_TX_PIN              PC2
+#define UART8_RX_PIN              PC3
+
+#define SERIAL_PORT_COUNT         8
+
+#define USE_SPI
+
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN              PA5
+#define SPI1_MISO_PIN             PA6
+#define SPI1_MOSI_PIN             PA7
+
+#define USE_SPI_DEVICE_2
+#define SPI2_SCK_PIN              PB13
+#define SPI2_MISO_PIN             PB14
+#define SPI2_MOSI_PIN             PB15
+
+#define USE_SPI_DEVICE_3
+#define SPI3_SCK_PIN              PC10
+#define SPI3_MISO_PIN             PC11
+#define SPI3_MOSI_PIN             PC12
+
+#define USE_IMU_MPU6000
+#define IMU_MPU6000_ALIGN         CW270_DEG_FLIP
```

Comment:
This appears incorrect. Pro and fs end up with different default orientation, and pro ends up with the front being the usb connector.

Reply:
For reference, the freestyle has the esc connector up front.

Reply:
fixed as discussed on discord, squashed and rebased against master.